lib/patch_ruby/api/orders_api.rb

```diff
@@ -161,20 +161,22 @@ def create_order_with_http_info(create_order_request, opts = {})
     end
 
     # Place an order
-    # Placing an order confirms an order's allocation of offsets. Only orders that are in the `draft` state can be placed 
-    # @param id [String] 
+    # Placing an order confirms an order's allocation of offsets. Only orders that are in the `draft` state can be placed
+    # @param id [String]
     # @param [Hash] opts the optional parameters
+    # @option opts [PlaceOrderRequest] :place_order_request
     # @return [OrderResponse]
-    def place_order(id, opts = {})
-    
-      data, _status_code, _headers = place_order_with_http_info(id, opts)
+    def place_order(id, place_order_request = {}, opts = {})
+      _place_order_request = Patch::PlaceOrderRequest.new(place_order_request)
+      data, _status_code, _headers = place_order_with_http_info(id, opts.merge!({place_order_request: place_order_request}))
```

Comment:
@holtbp to confirm, because I think it is the crux of the weird shit I'm trying to debug: did you by any chance add this `.merge!` manually?